sql/core/src/main/scala/org/apache/spark/sql/sources/ddl.scala

```diff
@@ -291,7 +291,7 @@ private[sql] case class ResolvedDataSource(provider: Class[_], relation: BaseRel
  * Returned for the "DESCRIBE [EXTENDED] [dbName.]tableName" command.
  * @param table The table to be described.
  * @param isExtended True if "DESCRIBE EXTENDED" is used. Otherwise, false.
- *                   It is effective only when the table is a Hive table.
+ * It is effective only when the table is a Hive table.
```

Comment:
I hate to nit-pick on a trivial change, but please undo this indentation change. From what I see in the rest of the code's scaladoc, continuations are aligned as it was, not wrapped all the way to the left. Have a look at `RDD.scala` for a lot of examples.